.test-infra/jenkins/PythonTestProperties.groovy

```diff
@@ -20,10 +20,10 @@ class PythonTestProperties {
   // Indicates all supported Python versions.
   // This must be sorted in ascending order.
   final static List<String> ALL_SUPPORTED_VERSIONS = [
-    '3.8',
     '3.9',
```

Comment:
is this file still being used given that we don't use Jenkins?

Reply:
I started to update the jenkins files and stopped once I realized what they were, it's worth doing some cleanup here eventually